### PR TITLE
Template SEO Optimizations

### PR DESCRIPTION
**Pull request by zchase**
Fixes: https://github.com/pulumi/marketing/issues/659, https://github.com/pulumi/marketing/issues/658, https://github.com/pulumi/marketing/issues/657, https://github.com/pulumi/marketing/issues/656, https://github.com/pulumi/marketing/issues/654, https://github.com/pulumi/marketing/issues/653

### CODE REVIEW

File: themes/default/content/templates/serverless-application/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Serverless Templates
+title: Serverless Applications and Templates
```
Comment (scottslowe) [inline themes/default/content/templates/serverless-application/_index.md:2]: I think "Serverless Application Templates" (omit the "and") is more accurate here

File: themes/default/content/templates/static-website/gcp/index.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
+title_tag: Learn to Host a Google Cloud Static Website
 title: Google Cloud Static Website
+h1: Host a Google Cloud Static Website
```
Comment (scottslowe) [inline themes/default/content/templates/static-website/gcp/index.md:4]: "Host a Static Website on Google Cloud" has a better ring to me

File: themes/default/content/templates/static-website/gcp/index.md [hunk 3]
```diff
@@ -22,7 +24,7 @@ The Google Cloud Static Website template creates an infrastructure as code proje
 
 ![An architecture diagram of the Pulumi Google Cloud Static Website template](./architecture.png)
 
-## Using this template
+## Using this template to host a Google Cloud Static Website
```
Comment (scottslowe) [inline themes/default/content/templates/static-website/gcp/index.md:27]: Same as previous (I'd prefer "Host a Static Website on Google Cloud"), if that will still get the same SEO optimizations

**Review by scottslowe — CHANGES_REQUESTED** [review]

File: themes/default/content/templates/static-website/gcp/index.md (hunk 3 above)
Comment (cnunciato) [reply themes/default/content/templates/static-website/gcp/index.md:27]: Keep in mind changes like these have a negative impact the right nav:

image

Will take a closer look at these changes today, but in general would prefer to be conservative and strike a balance between SEO and usability/consistency.

**Comment by cnunciato** [comment]
@zchase Just checking in -- still working on this one?

**Comment by zchase** [comment]
@cnunciato @scottslowe Pushed a commit to address your feedback. Let me know if anything else needs to be addressed. Thanks!

File: themes/default/content/templates/static-website/_index.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ weight: 1
 
 ### What are static websites?
 
-Static websites are an efficient and scalable way of building and serving websites created with HTML, CSS, and JavaScript. Popular frameworks for static websites include Next.js, Hugo, Gatsby, Jekyll, and more.
+A static website is an efficient and scalable way of building and serving websites created with HTML, CSS, and JavaScript. Popular frameworks for static websites include Next.js, Hugo, Gatsby, Jekyll, and more. On the other hand, a dynamic website uses frameworks like PHP, Node.js, and Go to render the webpage a the time of the request.
```
Comment (scottslowe) [inline themes/default/content/templates/static-website/_index.md:13]:
```suggestion
A static website is an efficient and scalable way of building and serving websites created with HTML, CSS, and JavaScript. Popular frameworks for static websites include Next.js, Hugo, Gatsby, Jekyll, and more. On the other hand, a dynamic website uses frameworks like PHP, Node.js, and Go to render the webpage at the time of the request.
```

**Review by scottslowe — APPROVED** [review]
One very small change (just a typo); otherwise it looks good. I'll have to rely on @cnunciato to understand the impact of the changes in the HTML snippet at the bottom. :-) Approving so you are unblocked.

**Pull request closed without merging**